components/notebook-crd/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Build the manager binary
+FROM golang:1.11.2 as builder
```

Comment:
you may want to make this an arg set by the Makefile

```
ARG GOLANG_VERSION=1.11.2
FROM golang:${GOLANG_VERSION} as builder
```

Reply:
done